tensorflow/lite/micro/micro_graph.cc

```diff
@@ -94,7 +101,14 @@ TfLiteStatus MicroGraph::PrepareSubgraphs() {
     current_subgraph_index_ = subgraph_idx;
 
     const SubGraph* subgraph = (*subgraphs_)[subgraph_idx];
-    for (size_t i = 0; i < subgraph->operators()->size(); ++i) {
```

Comment:
Do you think we can split this repeated code into a helper method?

Reply:
Done